pages/_app.tsx

```diff
@@ -1,16 +1,23 @@
-import '@/styles/globals.css'
-import type { AppProps } from 'next/app'
-import { Montserrat } from '@next/font/google'
+import "@/styles/globals.css";
+import type { AppProps } from "next/app";
+import { Cormorant_Garamond, Montserrat } from "@next/font/google";
 
 const montserrat = Montserrat({
-  subsets: ['latin'],
-  variable: '--font-montserrat'
-})
+  subsets: ["latin"],
+  variable: "--font-montserrat",
+});
+
+const garamond = Cormorant_Garamond({
+  subsets: ["latin-ext"],
+  style: "normal",
+  variable: "--font-garamond",
+  weight: "500",
+});
 
 export default function App({ Component, pageProps }: AppProps) {
   return (
-    <main className={`${montserrat.variable} font-sans`}>
+    <main className={`${montserrat.variable} ${garamond.variable} font-sans`}>
```

Comment:
imo lagi gi use in the same time ang font? walay way nga i pick lang for the numbers? kay kani ga cascade raba ang style so ma apply sa everything

Reply:
aahh ic ic, ako gi think kay needed ibutang dira ang font para maka create kog new tailwind utility class

Reply:
actually not sure sad, what did documentation say diay?

Reply:
hmm what if dili nalang nako i himo ug utility class ang instead import nalang in the component itself?

Reply:
when using tailwind kay in ana on. I prop drill ang font.variable para maka make ug utility class. pero yea we can also just not create a new utility class raman sad then i use ra siya sa component an dayun

Reply:
oks ra diay drian ka `font-garamond` man diay ka, sorry wa ko kita hahaha, resolved